.github/workflows/test.yml

```diff
@@ -13,8 +13,8 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - 18.0.0
-          - 19
+          - 20
+          - 21
```

Comment:
since 20 will always use the latest version within that range, i always also include a matrix item for the lower bound of the range as well. i'd recommend including 20.8.1 here so that we avoid unintentionally breaking a version that is claimed to be supported

Reply:
Good point! Adapted in https://github.com/semantic-release/gitlab/pull/647/commits/52c26906d7bb1772664a0e6508c3e1205868426f